functions/_tide_item_nvm.fish

```diff
@@ -0,0 +1,11 @@
+function _tide_item_nvm
+    set -l node_version (nvm current 2> /dev/null)
```

Comment:
unfortunately `nvm current 2>/dev/null` does not prevent fish from complaining and spamming the user with errors if an nvm command is not available. Add a `test functions -q nvm` instead of 2>/dev/null to prevent these errors.

Reply:
Done. The check would look like `(functions -q nvm; and nvm current)`